resolwe/flow/executors/run.py

```diff
@@ -8,6 +8,7 @@
     :members:
```

Comment:
Can you change the commit message to also mention that this improves log messages in the executor?

Also, add a CHANGELOG entry that can/will be extended in the next commits.